Source/Evolve/WarpXEvolve.cpp

```diff
@@ -403,11 +407,13 @@ WarpX::OneStep_nosub (Real cur_time)
     // the actual current J. This is computed later in WarpX::PushPSATD, by calling
     // WarpX::PSATDVayDeposition. The function SyncCurrent is called after that,
     // instead of here, so that we synchronize the correct current.
+    WARPX_PROFILE_REGION_START("WarpX::nosub::SyncJ_rho");
```

Comment:
Directly in `SyncCurrent` and `SyncRho` as `WARPX_PROFILE`?

Reply:
In could. Same as above.

(Apparently, while switching focus between windows, I clicked right on the "resolve" button, so don't take that as any commentary. 😄)